datastream-kafka/src/test/java/com/linkedin/datastream/kafka/TestKafkaPassthroughRecordMagicConverter.java

```diff
@@ -0,0 +1,38 @@
+/**
+ *  Copyright 2020 LinkedIn Corporation. All rights reserved.
+ *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
+ *  See the NOTICE file in the project root for additional information regarding copyright ownership.
+ */
+package com.linkedin.datastream.kafka;
+
+import org.apache.kafka.common.record.RecordBatch;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/**
+ * Tests for {@link KafkaPassthroughRecordMagicConverter}.
+ */
+@Test
+public class TestKafkaPassthroughRecordMagicConverter {
+
+  @Test
+  public void testMagicConversion() {
+    String magicValue0 = KafkaPassthroughRecordMagicConverter.convertMagicToString(RecordBatch.MAGIC_VALUE_V0);
```

Comment:
nit and feel free to ignore, but this one might read better as a for loop.

Reply:
done